packages/server/lib/fixture.js

```diff
@@ -60,7 +62,14 @@ module.exports = {
         if (matches.length === 0) {
           const relativePath = path.relative('.', p)
 
-          errors.throw('FIXTURE_NOT_FOUND', relativePath, extensions)
+          // TODO: there's no reason this error should be in
+          // the @packages/error list, it should be written in
+          // the driver since this error can only occur within
+          // driver commands and not outside of the test runner
+          const err = errors.get('FIXTURE_NOT_FOUND', relativePath, extensions)
```

Comment:
1. Would it make more sense to update the error to remove the ansi from the [FIXTURE_NOT_FOUND error](https://github.com/cypress-io/cypress/blob/develop/packages/errors/src/errors.ts#L894) (it has a `// fix me` comment we can remove).



2. I assume there is risk other errors created in the server that's sent back tot he driver will also need to strip out asni? It seem like it'd be safer to add this logic to the  reporter directly in the error component: https://github.com/cypress-io/cypress/blob/develop/packages/reporter/src/errors/test-error.tsx#L73

Reply:
That's what I was double checking - there is rarely a situation where we construct the error message in the server with ANSI styling AND send it through to the driver.

That's why I added the FIXME because there's no reason for the server to construct this error. The vast majority of the errors are formatting in the driver in this situation. That's why this broke to begin with. It's like a square peg in a round hole.

I looked into moving this fix more broadly into the server - but because this isn't actually the pattern we want to replicate it seemed fixing it only in this 1 place was most prudent. Once this error is moved into the driver it would obviate a need to ever need to do this in the server.

Reply:
Sounds good. Sounds like an easy fix that just needs to get done.